plugins/active_users/tests.js

```diff
@@ -0,0 +1,85 @@
+var request = require('supertest');
+var should = require('should');
+var testUtils = require("../../test/testUtils");
+request = request.agent(testUtils.url);
+
+var APP_KEY = "";
+var API_KEY_ADMIN = "";
+var APP_ID = "";
+
+describe('Testing Active Users', function() {
+    describe('Reset app', function() {
```

Comment:
Resetting should be done as almost last one.
You call reset. And then in the next step make sure it is reseted